arbiter-core/src/middleware.rs

```diff
@@ -308,7 +313,13 @@ impl Middleware for RevmMiddleware {
             _gas_refunded: _,
             logs,
             output,
-        } = unpack_execution_result(revm_result.result)?;
+        } = match unpack_execution_result(revm_result.result) {
```

Comment:
Nice i was wondering about this when i was debugging my issue

Reply:
Oh shit. I don't actually want to do it this way. Doing it with `?` is better.